test/unit/vc2013/unit.vcxproj

```diff
@@ -271,7 +270,8 @@
     <ClCompile Include="..\src\SystemTest.cpp" />
     <ClCompile Include="..\src\TestMain.cpp" />
     <ClCompile Include="..\src\UnicodeTest.cpp" />
-    <ClCompile Include="..\src\Utilities.cpp" />
+    <ClCompile Include="..\src\PolyLineTest.cpp" />

```

Comment:
Hm, just noticed that this somehow removed Utilities.cpp from the unit.vcxproj file and .filters, perhaps when resolving a merge conflict?